modules/host/persist.go

```diff
@@ -209,7 +209,7 @@ func (h *Host) load() error {
 func (h *Host) loadCompat(p *persistence) error {
 	var compatPersistence struct {
 		FinancialMetrics struct {
-			PotentialStorageRevenue types.Currency `json:"potentialerevenue"`
+			PotentialStorageRevenue types.Currency `json:"potentialrevenue"`
```

Comment:
this is actually not a typo, this one is here on purpose as compatibility code to a previous typo.